projects/go-lib/src/lib/components/go-datepicker/go-calendar.component.ts

```diff
@@ -121,5 +122,25 @@ export class GoCalendarComponent implements OnDestroy, OnInit {
     }
     this.currentMonth = this.selectedDate.getMonth();
     this.updateYear(this.selectedDate.getFullYear());
+    if (this.appendTo === 'body') {
+      setTimeout(() => {
+        const calenderBody: any = document.getElementsByClassName(
+          'append-body'
+        )[0];
+        const calenderBodyPosition: any = document
+          .querySelector('.append-body')
```

Comment:
If we change the above to be a `ViewChild` reference we can remove this `querySelector` and call `getBoundingClientRect` on the `ViewChild` element.